src/boxplot.jl

```diff
@@ -118,3 +118,47 @@ notch_width(q2, q4, N) = 1.58 * (q4-q2)/sqrt(N)
     ()
 end
 Plots.@deps boxplot shape scatter
+
+
+# ------------------------------------------------------------------------------
+# Grouped Boxplot
+
+@userplot GroupedBoxplot
+
+recipetype(::Val{:groupedboxplot}, args...) = GroupedBoxplot(args)
+
+@recipe function f(g::GroupedBoxplot; spacing = 0.1)
+    x, y = grouped_xy(g.args...)
+
+    # extract xnums and set default bar width.
+    # might need to set xticks as well
+    x = if eltype(x) <: Number
+		bar_width --> (0.8 * mean(diff(x)))
```

Comment:
Indentation is a bit strange here and in the rest of the diff.

Reply:
hopefully fixed